### PR TITLE
fix: Unable to open old executions

### PR DESCRIPTION
**Pull request by PunkSage**
Resolves: https://github.com/kubeshop/testkube/issues/2962

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = {
     'react-hooks/rules-of-hooks': 'error', // Checks rules of Hooks
     'no-console': 'warn',
     'no-undef': 'off',
-    semi: 'error',
```
Comment (PunkSage) [inline .eslintrc.js:21]: All of these problems that allow for proper compilation should not prevent a developer from seeing the results of the changes

File: src/components/organisms/EntityDetails/EntityDetailsContainer/EntityDetailsContainer.tsx [hunk 2]
```diff
@@ -177,28 +177,15 @@ const EntityDetailsContainer: React.FC<EntityDetailsBlueprint> = props => {
   };
 
   useEffect(() => {
-    if (execId && executionsList && executionsList?.results) {
```
Comment (Toxer10) [inline src/components/organisms/EntityDetails/EntityDetailsContainer/EntityDetailsContainer.tsx:180]: Does this feature work without this code?

File: .eslintrc.js (hunk 1 above)
Comment (Toxer10) [reply .eslintrc.js:21]: I would not want to have ```warn``` on this rule, it's too crucial to be omitted ocassionally

**Review by Toxer10 — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (PunkSage) [reply .eslintrc.js:21]: All of these issues could be marked as warnings and be fixed when you compile and push-out the code to the repository. Throwing this as an error breaks the developers feedback loop and disallow you do concentrate on the root of the problem you’re trying to solve. After a few changes of the code you are forced to fight with the linter.

File: src/components/organisms/EntityDetails/EntityDetailsContainer/EntityDetailsContainer.tsx (hunk 2 above)
Comment (PunkSage) [reply src/components/organisms/EntityDetails/EntityDetailsContainer/EntityDetailsContainer.tsx:180]: I cannot find any feature that would be broken after removal of this code. If you can verify this I would greatly appreciate.

File: src/components/organisms/EntityDetails/EntityDetailsContainer/EntityDetailsContainer.tsx [hunk 3]
```diff
@@ -177,28 +177,24 @@ const EntityDetailsContainer: React.FC<EntityDetailsBlueprint> = props => {
...
+      const indexOfDisplayedExecution = executionDetails ? executionsList.results?.indexOf(executionDetails) + 1 : null;
+      if (!indexOfDisplayedExecution) setDaysFilterValue(0);
+    }
+  }, [params, executionsList]);
 
-        const targetEntityArrayIndex = executionsList.results?.indexOf(targetEntity[0]) + 1;
+  useEffect(() => {
+    const executionDetails = executionsList?.results?.find((execution: any) => execution.id === execId);
+    const indexOfDisplayedExecution = executionDetails ? executionsList.results?.indexOf(executionDetails) + 1 : null;
+    if (indexOfDisplayedExecution) setCurrentPage(Math.ceil(indexOfDisplayedExecution / 10));
```
Comment (pavloburchak) [inline src/components/organisms/EntityDetails/EntityDetailsContainer/EntityDetailsContainer.tsx:190]: Here the same as above

**Review by pavloburchak — APPROVED** [review]

**Pull request merged**